SwiftUIKit-Example/SwiftUIKit-Example/Views/HomeScreenVC.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  BasicVC.swift
+//  SwiftUIKit-Example
+//
+//  Created by finos.son.le on 22/04/2021.
+//
+import UIKit
+
+import SwiftUIKit
+struct Item {
```

Comment:
## Convention

Consider namespace for naming type (struct/enum/class) or property.
There are many types with the name `Item` that can be used in different places. If this `Item` just only use for Home Screen, put it inside `HomeScreenVC` class scope